admin/page-assets-metabox.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * Handles creating the page assets metaboxes
+ **/
+if ( ! class_exists( 'UCF_Page_Assets_Metabox' ) ) {
+    class UCF_Page_Assets_Metabox {
+
+		/**
+		 * Enqeues admin assets
+		 * @author Jim Barnes
+		 * @since 1.0.0
+		 * @param $hook string | The current admin hook
+		 **/
+		public static function enqueue_assets( $hook ) {
+			if ( 'post.php' === $hook ) {
+				wp_enqueue_script(
+					'ucf-page-assets-js',
+					UCF_PAGE_ASSETS__JS_URL . '/ucf-page-assets.min.js',
+					array( 'jquery' ),
+					null,
+					true
+				);
+			}
+		}
+
+		/**
+		 * Adds the Custom Page Assets metabox
+		 * @author Jim Barnes
+		 * @since 1.0.0
+		 **/
+		public static function add_meta_box() {
+			add_meta_box(
+				'ucf-page-assets',
+				__( 'Custom Page Assets' ),
+				array( 'UCF_Page_Assets_Metabox', 'metabox_markup' ),
+				UCF_Page_Assets_Config::enabled_posts()	
+			);
+		}
+        
+		/**
+		 * The markup callback for the Custom Page Assets metabox
+		 * @author Jim Barnes
+		 * @since 1.0.0
+		 * @param $post WP_Post | The current post object
+		 * @return string | The function output is echoed
+		 **/
+        public static function metabox_markup( $post ) {
+            wp_nonce_field(  'ucf_page_assets_nonce_save', 'ucf_page_assets_nonce' );
+			$upload_link = esc_url( get_upload_iframe_src( 'media', $post->ID ) );
```

Comment:
There's some funky tabbing happening in a few places--maybe force tabs in your text editor and re-save these files?